doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -330,6 +346,8 @@ the type of :token:`term` to be :token:`type`.
 Inferable subterms
 ------------------
 
+.. index:: _
```

Comment:
This does not appear in the symbol section of the index but in a new `_` section. Not great but not a critical problem either.